src/main/resources/static/characterCreator.js

```diff
@@ -0,0 +1,42 @@
+function sendAjaxRequest(heroClass) {
+    sendAjaxArmorRequest(heroClass);
+    sendAjaxWeaponRequest(heroClass);
+}
+
+function sendAjaxArmorRequest(heroClass) {
+    fetch(`http://localhost:8080/charClassToArmor`, { method: 'get' })
+        .then(function (response) {
+            return response.json();
+        }).then(async function (data) {
+            data.forEach(e => {
+                let firstSelect = document.getElementById('selectArmor' + e);
+                firstSelect.disabled = true;
+            });
+            const response = await fetch(`http://localhost:8080/charClassToArmor/` + heroClass);
+            const result = await response.json();
+            console.log(result);
+            result.forEach(elem => {
+                let chainedSelect = document.getElementById('selectArmor' + elem);
+                chainedSelect.disabled = false;
+            });;
+        })
+}
+
+function sendAjaxWeaponRequest(heroClass) {
+    fetch(`http://localhost:8080/charClassToWeapon`, { method: 'get' })
+        .then(function (response) {
+            return response.json();
+        }).then(async function (data) {
+            data.forEach(e => {
+                let firstSelect = document.getElementById('selectWeapon' + e);
+                firstSelect.disabled = true;
+            });
+            const response = await fetch(`http://localhost:8080/charClassToWeapon/` + heroClass);
+            const result = await response.json();
+            console.log(result);
+            result.forEach(elem => {
+                let chainedSelect = document.getElementById('selectWeapon' + elem);
+                chainedSelect.disabled = false;
+            });;
+        })
+}
```

Comment:
Tu nic nie rozumiem :P no ale to tak jak pisaliśmy to JS ;)

Reply:
Jasne, chociaż mogę opowiedzieć. 
`fetch(`http://localhost:8080/charClassToArmor`, { method: 'get' })`
strzela zapytaniem do restowego api, które utworzyłem w klasie CharacterCreatorRestController. Jak z chroma wejdziesz na taki adres,, to zobaczysz co wyrzuca. Biorę te parametry spod tego adresu i dezaktywuję wszystkie buttony o tych ID, po czym strzelam na restowe API 
`fetch(`http://localhost:8080/charClassToWeapon/` + heroClass);`
czyli takie same, tylko z nazwą klasy po slashu i odblokowuję wszystkie buttony o tych ID, które zwróci mi to API.
Potestuj, to zobaczysz jak to działa